dcs/task.py

```diff
@@ -1077,6 +1082,39 @@ def __eq__(self, other) -> bool:
         return False
 
 
+class CarpetBombing(Task):
+    Id = "CarpetBombing"
+
+    class AttackType(Enum):
+        Carpet = "Carpet",
+
+        def __eq__(self, other: Any) -> bool:
+            if isinstance(other, str):
+                return self.value == other
+            return self == other
+
+    def __init__(self, altitude: int = 2000, altitude_enabled: bool = False,
```

Comment:
`altitude` is a misleading name. It's not the altitude for the attack, it's the minimum allowed altitude.

Replace `altitude` and `altitude_enabled` with a single `minimum_attack_altitude: Optional[int] = None`

Reply:
@DanAlbert - I understand and thinking about it now (after understanding more about DCS/miz/pydcs) I think ME should be preferred since ME user is only capable of changing the ME things and they should be named following ME naming convention too, since pydcs caller is a user emulating ME and object parameters in pydcs are the ME parameters. I'll refactor this.